modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/I18n/Language.json

```diff
@@ -443,6 +444,7 @@
 	"this-field-is-required": "This field is required.",
 	"this-key-does-not-expire": "This key does not expire",
 	"this-may-affect-your-use-of-the-customer-portal-please-go-to": "this may affect your use of the Customer Portal, please go to",
+	"this-team-member-is-assigned-as-an-Incident-contact-and-cannot-be-removed": "This team member is assigned as an Incident Contact and cannot be removed. Please remove them from all incident contact roles and then try again.",
```

Comment:
`Incident` should not be capitalized in the key, can you please lowercase it? As well as update where the key is used in the code.

Also in the translation, `Incident Contact` is capitalized in the first sentence, and lowercase in the second sentence. Is this correct? Is this how they wanted it?

Reply:
Ok!! in the acceptance criteria, the sentence looks like this: "This team member is assigned as an Incident Contact and cannot be removed. Please remove them from all incident contact roles and then try again."
would it be better to correct it and make it uppercase?

Reply:
Okay, if the acceptance criteria has it like that then I think it is okay to leave it. So just the language key needs to be changed then. Thanks!

Reply:
Done!!